src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -31,6 +31,21 @@ type Props = {
   titlePostfix: string,
 };
 
+const getPaginatedTitle = (
```

Comment:
Looks great! My only nit is that this method name returns an  object, not just the title string, so a name like `getPageById` might be better than `getPaginatedTitle`.

Reply:
I like this method name much better. Renamed in last commit. Thanks!